theme/style.css

```diff
@@ -35,13 +35,14 @@ img {
   text-transform: uppercase;
   font-family: monospace;
   font-size: 3em;
-  color: hotpink;
+  color: fuchsia;
   text-shadow: -1px -1px 0 #000,
   1px -1px 0 #000,
   -1px 1px 0 #000,
   1px 1px 0 #000;
   letter-spacing: 0.3em;
   border-bottom: 3px #000 dotted;
+  margin-top: -10px;
```

Comment:
you should _rarely_ need a negative `margin`. Consider modifying the `margin` or the `padding` of the other elements or the container instead!